edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/DeviceScopeAuthenticator.cs

```diff
@@ -138,7 +138,7 @@ async Task<(bool isAuthenticated, bool serviceIdentityFound)> AuthenticateWithAu
             Option<string> authChain = await this.deviceScopeIdentitiesCache.GetAuthChain(credentials.Identity.Id);
```

Comment:
I found that argument syncServiceIdentity is not used.  Do we need it?

Reply:
that's a bug, it should actually get passed down to AuthenticateWithServiceIdentity.  Good catch =)

---
In reply to: [458489677](https://github.com/Azure/iotedge/pull/3248#discussion_r458489677) [](ancestors = 458489677)